sdk/core/azure-core-amqp/inc/azure/core/amqp/message_receiver.hpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-Licence-Identifier: MIT
+
+#pragma once
+
+#include "claims_based_security.hpp"
+#include "common/async_operation_queue.hpp"
+#include "connection_string_credential.hpp"
+#include "link.hpp"
+#include "models/amqp_message.hpp"
+#include "models/amqp_value.hpp"
+#include "session.hpp"
+#include <azure/core/credentials/credentials.hpp>
+#include <azure/core/nullable.hpp>
+#include <vector>
```

Comment:
Same formatting nit here/elsewhere to separate include blocks which makes clang-format sort it easier.

Reply:
Same response as before - if we want to define an official ordering for headers, we should define it in the clang-format file, rather than depending on ad-hoc authoring rules.